org.garudalinux.firedragon.appdata.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.garudalinux.firedragon</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MPL-2.0 AND LicenseRef-proprietary=https://raw.githubusercontent.com/Floorp-Projects/Floorp-private-components/main/LICENSE</project_license>
+  <name>FireDragon</name>
+  <summary>Floorp fork build using custom branding &amp; settings</summary>
+  <developer_name>FireDragon Team</developer_name>
+  <icon type="stock">org.garudalinux.firedragon</icon>
```

Comment:
```suggestion
```

Not needed for desktop applications

Reply:
Fixed in 37c1c0b58ab2d61b83abc0154a2710001f36a07c